src/views/Graph.js

```diff
@@ -107,7 +110,7 @@ var GraphView = Backbone.View.extend({
         // Kick off the render loop
         this.animate();
 
-        this.listenTo(events, 'datalasso:data:new', this.redrawEverything);
+        this.listenTo(store, 'change:entries change:mappings change:scales change:attributes change:selectedEntries', this.redrawEverything);
```

Comment:
can several of these events fire at the same time? might want a debounced `this.redrawEverything` in that case